CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 1.12.1 (2021-01-21)
+
+-   Improved syntax highlighting for:
+    -   text granules
+    -   tags in text granules
+    -   function calls in function call arguments
```

Comment:
Do you mean something like…

```suggestion
    -   function calls nested in another function’s call
```

Not perfect either; maybe there’s something better.